### DISCUSSION TITLE
How to notify all users of the change to /command

### DISCUSSION THREAD

**Discussion by TAOhtani** [discordjs/discord.js#8687] [Q&A] [answered]
Nice to meet you. I have been running a bot that I created as a hobby for a while now, and I am trying to support /command, though I am late to the game.
I used to use "!command" is now "/command", what is the recommended way to send a message to everyone who has installed the bot?

We currently have over 9000 servers installed and are concerned about rate limiting issues.

**Comment by almostSouji** [comment, marked as answer]
If you have an announcement channel in your bot home/hub/dev server that people follow, that'd be preferable.
Sending a message to semi-random channels in many guilds is not something we'd recommend.

Discord themselves have implemented command suggestions, so the slash command should come up for people, even if they use `!` as a prefix (unless they manually disable that suggestions via right-click on the chat box

![image](https://user-images.githubusercontent.com/26532370/193213592-5222af7d-6c9d-466a-b5f1-c6b2bb23a3cf.png)

**Reply by TAOhtani** [reply]
Thanks for the answer.

Unfortunately, I am the sole developer and had no way to notify you.

As you showed in the image, I used the ! command to get a /command suggestion was a blind spot. I too think this is mostly sufficient.

Thank you for the help.
